addons/autoLock/autoLock.lua

```diff
@@ -0,0 +1,166 @@
+-- 
+-- autoLock v1.0
+-- 
+-- Copyright ©2015, bangerang
+-- All rights reserved.
+-- 
+-- Redistribution and use in source and binary forms, with or without
+-- modification, are permitted provided that the following conditions are met:
+-- 
+-- * Redistributions of source code must retain the above copyright
+--   notice, this list of conditions and the following disclaimer.
+-- * Redistributions in binary form must reproduce the above copyright
+--   notice, this list of conditions and the following disclaimer in the
+--   documentation and/or other materials provided with the distribution.
+-- * Neither the name of autoLock nor the names of its contributors may be 
+--   used to endorse or promote products derived from this software without
+--   specific prior written permission.
+
+-- THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+-- ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+-- WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+-- DISCLAIMED. IN NO EVENT SHALL ReaperX BE LIABLE FOR ANY
+-- DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+-- (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+-- LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+-- ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+-- (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+-- SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+--
+--
+-- Automatically equips a gear set and uses the /lockstyle command when leaving town.
+-- This addon requires the Gearswap addon.
+--
+-- To use this addon create a gear set titled sets.lockstyle in your job.lua file.
+-- Ex:
+-- 		sets.lockstyle = {
+--			head="Raider's Bonnet +2",
+--			body="Raider's Vest +2",hands="Raider's Armlets +2",
+--			legs="Raider's Culottes +2",feet="Raider's Poulaines +2"}
+--
+--
+
+_addon.name = "autoLock"
+_addon.author = "bangerang"
+_addon.version = "1.0"
+_addon.commands = {'al', 'autolock'}
+_addon.language = 'english'
+
+require('luau')
+config = require('config')
+
+defaults = {}
+defaults.zoneDelay = 10
```

Comment:
This is not mandatory, but we either use `snake_case` or `PascalCase` for settings values, the latter being preferred. Up to you if you wanna change it, but `camelCase` goes against conventions and it makes me sad :'(

Same goes for the addon name.